docs/source/manim_directive.py

```diff
@@ -276,18 +277,20 @@ def setup(app):
 {% if not (save_as_gif or save_last_frame) %}
 .. raw:: html
 
-    <video class="manim-video" controls loop autoplay src="./{{ output_file }}.mp4"></video>
+    <video id="{{ clsname_lowercase }}" class="manim-video" controls loop autoplay src="./{{ output_file }}.mp4"></video>
 {% elif save_as_gif %}
 .. image:: /{{ filesrc_rel }}
     :align: center
+    :name: {{ clsname_lowercase }}
 {% elif save_last_frame %}
 .. image:: /{{ filesrc_rel }}
     :align: center
+    :name: {{ clsname_lowercase }}
 {% endif %}
 {% if not hide_source %}
 .. raw:: html
 
-    <div class="example-header">{{ clsname }}</div>
+    <h5 class="example-header">{{ clsname }}<a class="headerlink" href="#{{ clsname_lowercase }}">¶</a></h5>
```

Comment:
I'd rather the anchor go to the title, but the video also works I guess.

Reply:
I found to the video/image a bit more intuitive, then you don't have to scroll up before seeing what the example is about. :-)